playground/README.md

```diff
@@ -0,0 +1,8 @@
+# LWC playground
+
+The simplest LWC setup for simple experiments and bug reproductions.
+
+```sh
+$ npm run dev       # Get app server running
+$ npm run build     # Build app in production mode
```

Comment:
I'm aware that we plan on migrating away from yarn but I think it would be worth it to keep things consistent within the repository.
```suggestion
$ yarn dev       # Get app server running
$ yarn build     # Build app in production mode
```